### DISCUSSION TITLE
redlock-algorithm/

### DISCUSSION THREAD

**Discussion by giscus** [devHudi/hudi.blog#226] [Comments]
# redlock-algorithm/

https://hudi.blog/redlock-algorithm/

**Comment by hgo641** [comment]
최고에요! 대단해요! 멋져요!

**Comment by msung99** [comment]
오늘도 한 수 배워갑니다!!!!

**Comment by ACACIANh** [comment]
분산시스템을 조금씩 공부중입니다
Redis 는 가용성을 중점에 둔 AP 시스템으로 C(일관성)이 보장되지 않아 보장하려면 결국 CP 시스템을 이용해야하는것으로 이해했습니다
그렇다면 해당 알고리즘은 AP 시스템이지만 일관성을 최대한 올려주는 기법으로 이해하고 넘어가면 적절할까요?
